lighthouse-core/audits/byte-efficiency/uses-responsive-images.js

```diff
@@ -48,28 +48,42 @@ class UsesResponsiveImages extends ByteEfficiencyAudit {
 
   /**
    * @param {LH.Artifacts.ImageElement} image
-   * @param {number} DPR devicePixelRatio
+   * @param {LH.Artifacts.ViewportDimensions} viewport
    * @return {null|Error|LH.Audit.ByteEfficiencyItem};
    */
-  static computeWaste(image, DPR) {
+  static computeWaste(image, viewport) {
```

Comment:
`ViewportDimensions`? I think we try to keep artifacts named LikeThis in other places, not sure if we do it everywhere tho

Reply:
sure, we definitely tend to as locals but I'd say it varies a little more once they get passed around. moving in that direction sgtm either way though 👍